src/main/java/net/fabricmc/loader/impl/game/GameProvider.java

```diff
@@ -38,6 +41,24 @@ public interface GameProvider { // name directly referenced in net.fabricmc.load
 	Path getLaunchDirectory();
 	boolean isObfuscated();
 	boolean requiresUrlClassLoader();
+	Set<BuiltinTransform> getBuiltinTransforms(String className);
```

Comment:
Should we make this a default method for backwards compat? Other games couldn't use transforms previously so an empty set would make sense. 

Doesn't really matter too much though.